tox.ini

```diff
@@ -9,10 +9,9 @@ envlist = py27,py33,py34,py35
 [testenv]
 # NB(sross): disabling E225,E226,E227,E901 make pep8 think Cython is ok
 # NB(sross): disable E402 because of the GSSAPI="blah" lines needed by Cython
```

Comment:
These comments should be moved to where they live now.  (They're duplicated there, so I think they should go away entirely.)

Reply:
ack, I just forgot to remove them.